fedimint-core/src/lib.rs

```diff
@@ -253,6 +255,20 @@ pub enum CoreError {
     MismatchingVariant(&'static str, &'static str),
 }
 
+// Encode features for a bolt11 invoice without encoding the length.
+// This functionality was available in `lightning` v0.0.123, but has since been
+// removed. See the original code here:
+// https://docs.rs/lightning/0.0.123/src/lightning/ln/features.rs.html#745-750
+// https://docs.rs/lightning/0.0.123/src/lightning/ln/features.rs.html#1008-1012
+pub fn encode_bolt11_invoice_features_without_length(features: &Bolt11InvoiceFeatures) -> Vec<u8> {
+    let mut feature_bytes = vec![];
+    for f in features.le_flags().iter().rev() {
+        f.write(&mut feature_bytes)
+            .expect("Writing to byte vec can't fail");
+    }
+    feature_bytes
+}
```

Comment:
Note for reviewer: Non-trivial change

Reply:
This is only used in tests, so no need to sweat? Or do I miss anything?